spec/integration_spec.rb

```diff
@@ -1,25 +1,19 @@
 
 context "Apivore tests running against a mock API" do
-  describe "valid swagger and correctly implemented API" do
-    it 'passes when the swagger description does match the implemented API' do
-      stdout = `rspec spec/data/example_specs.rb --example 'passing doc'`
-      expect(stdout).to match(/0 failures/)
-    end
-  end
 
-  describe "the swagger documents an unimplemented path" do
-    it 'should show an alert that the specific path is unimplemented' do
-      stdout = `rspec spec/data/example_specs.rb --example 'unimplemented path in doc'`
+  describe "the test checks undocumented path" do
+    it 'should show an alert that the specific path is undocumented' do
+      stdout = `rspec spec/data/example_specs.rb --example 'undocumented path in test'`
       expect(stdout).to match(/1 failure/)
-      expect(stdout).to match(/expected 200 array, got 404/)
+      expect(stdout).to match(/does not have a documented path/)
```

Comment:
The previous expectation of this test is the correct one  -- a successful 200 response containing an array is documented, but the mock API is returning a 404. 
`expect(stdout).to match(/expected 200 array, got 404/)`

The intent of this test was to be a clear indication that a route wasn't implemented.
Now that I think about it, this scenario should be covered by
*a path exists but the API responds with an unexpected http response code*

The existence  of the path or route doesn't really matter here. If that is the case, then this test can be removed, and the other scenario updated to:
*the API responds with an unexpected http response code* ? I'm thinking as I type here. Initially the intent was that an error in how a response was implemented was a different scenario to a path not existing at all..